ibis/backends/base/__init__.py

```diff
@@ -1433,9 +1433,14 @@ def _from_url(self, url: str, **kwargs) -> BaseBackend:
         url = urlparse(url)
         netloc = url.netloc
         parts = list(filter(None, (netloc, url.path[bool(netloc) :])))
-        database = (
-            Path(*parts).absolute() if parts and parts != [":memory:"] else ":memory:"
-        )
+        database = Path(*parts) if parts and parts != [":memory:"] else ":memory:"
+        if (strdatabase := str(database)).startswith("md:") or strdatabase.startswith(
```

Comment:
Should we set up a secret to allow testing this in CI?

Reply:
We could do.  I don't think they rotate the tokens particularly often and we've already broken it once.  I can put in my credentials -- I don't use motherduck except for testing and I'll confirm that I don't have any payment methods set up in there.

Reply:
last I checked you can't rotate your key :) have to request it be done on their end